src/main/java/frc/robot/subsystems/RoboState.java

```diff
@@ -0,0 +1,49 @@
+package frc.robot.subsystems;
```

Comment:
Why is this robot state class being included in the Lock on pull request?

Reply:
I might have made some changes to robot state without changing branches.